types/index.d.ts

```diff
@@ -572,9 +573,22 @@ declare namespace CbServer {
 
   interface TriggersAsync {
     create(option: Omit<TriggerCreateOptions, "system_key">): Promise<object>;
-    read(query: Query): Promise<object[]>;
-    update(query: Query, changes: object): Promise<object[]>;
-    delete(query: Query): Promise<object>;
+    read(query: AsyncPlatformQuery): Promise<object[]>;
+    update(query: AsyncPlatformQuery, changes: object): Promise<object[]>;
+    delete(query: AsyncPlatformQuery): Promise<object>;
+  }
+
+  interface TimersAsyncCreateOptions extends TimerCreateOptions {
+    name: string;
+    user_id?: string;
+    user_type?: number; // user type of user_id (1=dev, 2=user, 3=device)
```

Comment:
You could use a union for this - `user_type?: 1 | 2 | 3`

Reply:
Thank you